src/infra/postgres/client/client.go

```diff
@@ -45,6 +46,9 @@ func (c *PostgresClient) Insert(ctx context.Context, model ...interface{}) error
 
 func (c *PostgresClient) SelectPK(ctx context.Context, model ...interface{}) error {
 	err := c.db.ModelContext(ctx, model...).WherePK().Select()
+	if err == pg.ErrNoRows {
```

Comment:
You can remove this `if` statement as well. This is done by the `parseErrorResponse` function